README.md

```diff
@@ -77,7 +77,7 @@ optional arguments:
   --dump-path DUMP_PATH
                         Specify the path to dump to
   --dump-on DUMP_ON     Specify when to dump; can dump on: exit, transaction
-  --lite-mode           Applies lite-mode optimizations by disabling some features. (In the current version, lite-mode doesn't affect performance)
+  --lite-mode           Applies lite-mode optimizations by disabling some features (like transaction and block hash calculations).
```

Comment:
Let's change the this help text to be more transparent (should say what it does, without using the word "like") and non-recursive (shouldn't mention lite-mode in lite-mode explanation)

Reply:
true, updated